.github/workflows/maven.yml

```diff
@@ -40,7 +40,7 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-maven-
       - name: Build
-        run: ./mvnw --batch-mode -Pcoverage clean verify
+        run: ./mvnw --batch-mode -Pcoverage clean install
```

Comment:
We need the verify here to run integration tests

Reply:
`install` includes `verify` (see [Maven lifecycle](https://maven.apache.org/guides/introduction/introduction-to-the-lifecycle.html)), i.e. ITs are still being executed.

However, as I said in the PR description, since you've got no multi-module project, you can stick to `verify` I guess. (Should have split up the PR in the first place.)

Reply:
ah ok, I see what you meant. Yes we do not need to install any jars or something else in the local repository, because we only have one deployable here. :)

Reply:
Will remove the commit. 👍